res/css/views/dialogs/_JoinRuleDropdown.pcss

```diff
@@ -54,6 +54,11 @@ limitations under the License.
         padding: 1px;
     }
 
+    .mx_JoinRuleDropdown_knock::before {
+        mask-image: url("$(res)/img/element-icons/hand.svg");
+        mask-size: contain;
+    }
+
```

Comment:
As a heads up we should not use the mask technique anymore for icons, https://github.com/matrix-org/matrix-react-sdk/blob/develop/docs/icons.md